src/main/resources/data/retail-transactions-check-images.json

```diff
@@ -0,0 +1,32 @@
+[
+  {
+    "amount": 3.99,
+    "category": "Food Drinks",
+    "counterPartyCountry": "US",
+    "counterPartyName": "In N Out Burger",
+    "currency": "USD",
+    "reference": "czyo9c2n7z",
+    "type": "POS",
+    "typeGroup": "Payment",
+    "description": "IN-N-OUT BURGER #239AMERICAN FO",
```

Comment:
Should we mention in the description that this transaction should have check images to make it easier to find in the app?

Reply:
for whom do we need this description?

Reply:
Mobile developers and users looking at the transaction list to pick one that has check images.

Reply:
done

Reply:
Sorry, where? I can't find it.

Reply:
sorry just pushed it.